platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+| [Adoption](#adoption)       | _Why and how do users discover and use internal platforms and platform capabilities?_      | Erratic                | Extrinsic push        | Intrinsic pull         | Participatory                |
+| [Interfaces](#interfaces)   | _How do users interact with and consume platform capabilities?_                            | Custom processes       | Standard tooling      | Self-service solutions | Integrated services          |
+| [Operations](#operations)   | _How are platforms and their capabilities planned, prioritized, developed and maintained?_ | By request             | Centrally tracked     | Centrally enabled      | Managed services             |
+| [Measurement](#measurement) | _What is the process for gathering and incorporating feedback and learning?_               | Ad hoc                 | Consistent collection | Insights               | Quantitative and qualitative |
+
+## Model Details
+
+The aspects are:
+* [Investment](#investment): How are staff and funds allocated to platform capabilities?
+* [Adoption](#adoption): Why and how do users discover and use internal platforms and platform capabilities?
+* [Interfaces](#interfaces): How do users interact with and consume platform capabilities?
+* [Operations](#operations): How are platforms and their capabilities planned, prioritized, developed and maintained?
+* [Measurement](#measurement): What is the process for gathering and incorporating feedback and learning?
+
+### Investment
+<h4 style="color:gray"><i>How are staff and funds allocated to platform capabilities?</i></h4>
+
+Investment in platforms and platform engineering is the process of allocating budget and people to build and maintain common capabilities. It is common for initiatives to be described as organically built from the bottom up, or driven through top down initiatives. In either case, it is the ability to invest sustained effort that drives high-impact work. This aspect captures how the scale and breadth of investment can impact platform success.
+
+##### Level 1, Provisional - Voluntary or temporary
+
+Individual capabilities may exist to provide common foundations for common or critical functionality. These capabilities are built and maintained out of necessity rather than planned and intentional funding. 
+
+These capabilities are built and maintained by people assigned temporarily or voluntarily; no central funding or staffing are intentionally allocated to them. They depend on the current tactical requirements of their users.
+
+###### Characteristics:
+
+* Capabilities may be supported by individual employees rather than groups or teams as the original implementation was never adopted by a permanent team
+* Upgrades and improvements are rarely prioritized, and when they are they do not include migration plans for all users of the capability
+* Security patches are not tracked or managed, only being applied when there is a specific breach or industry wide concern
```

Comment:
Aren't these two characteristics of operations maturity?

Reply:
Have swapped out for two different ones that touch on the same ideas but with less of an operations tilt.